lib/tasks/add_lab_deadlines.rake

```diff
@@ -0,0 +1,8 @@
+task :add_lab_deadlines => :environment do
+  lab_names = ["Lab 1: Source Control", "Lab 2: Styling", "Lab 3: Client-Side Interactivity", "Lab 4: Servers", "Lab 5: Putting it together", "Lab 6: AJAX", "Lab 7: Databases", "Lab 8: Analytics"]
+  deadlines = ["2016/01/07 14:00:00", "2016/01/14 14:00:00", "2016/01/21 14:00:00", "2016/01/28 14:00:00", "2016/02/04 14:00:00", "2016/02/11 14:00:00", "2016/02/18 14:00:00", "2016/02/25 14:00:00", "2016/03/03 14:00:00", "2016/03/11 14:00:00"]
+  8.times.each do |i|
+    time = Time.parse(deadlines[i]+" UTC")
```

Comment:
The UTC would cause issue here, since the app is running under the Pacific time zone, if you do Time.now and compare with the deadline time, the deadline time would be -8 hours, you can just get rid of the `+ " UTC"` statement.